selfdrive/car/hyundai/carcontroller.py

```diff
@@ -108,7 +108,8 @@ def update(self, CC, CS):
     if self.frame % 5 == 0 and self.car_fingerprint in (CAR.SONATA, CAR.PALISADE, CAR.IONIQ, CAR.KIA_NIRO_EV, CAR.KIA_NIRO_HEV_2021,
                                                         CAR.IONIQ_EV_2020, CAR.IONIQ_PHEV, CAR.KIA_CEED, CAR.KIA_SELTOS, CAR.KONA_EV,
                                                         CAR.ELANTRA_2021, CAR.ELANTRA_HEV_2021, CAR.SONATA_HYBRID, CAR.KONA_HEV, CAR.SANTA_FE_2022,
-                                                        CAR.KIA_K5_2021, CAR.IONIQ_HEV_2022, CAR.SANTA_FE_HEV_2022, CAR.GENESIS_G70_2020, CAR.SANTA_FE_PHEV_2022):
+                                                        CAR.KIA_K5_2021, CAR.IONIQ_HEV_2022, CAR.SANTA_FE_HEV_2022, CAR.GENESIS_G70_2020,
+                                                        CAR.SANTA_FE_PHEV_2022, CAR.IONIQ_PHEV_2019):
```

Comment:
Are you sure this is required? The car isn't actually sending this message itself, whereas another car in this list, like the Sonata is sending this on bus 2

Reply:
Good catch! You're correct, this is not required.